src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -127,6 +128,35 @@ public void parse_viewOnly_success() throws ParseException, IOException {
                 OUTPUT_DIRECTORY_ABSOLUTE, ((ViewCliArguments) cliArguments).getReportDirectoryPath()));
     }
 
+    @Test
+    public void parse_withIgnore_success() throws ParseException {
+        String input = String.format("-repos \"%s\" %s --ignore-standalone-config",
+                TEST_REPO_REPOSENSE, TEST_REPO_DELTA);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof LocationsCliArguments);
+        Assert.assertTrue(((LocationsCliArguments) cliArguments).isStandaloneConfigIgnored());
+    }
+
+    @Test
+    public void parse_withIgnoreAlias_success() throws ParseException {
```

Comment:
Should merge `parse_withIgnore_success` and `parse_withIgnoreAlias_success` together so you can test
`Assert.assertEquals` on both of the `CliArguments`